components/webui/server/tsconfig.json

```
@@ -16,11 +16,6 @@
     "skipLibCheck": true,
     "target": "es2022",
```

```suggestion
    "target": "ES2022",
```

also above "lib" should be:
```
    "lib": [
      "ES2023"
    ],
```